### DISCUSSION TITLE
Message writing issue: No CTRL-Z after preview (loss of edit history)

### DISCUSSION THREAD

**Discussion by marckruzik** [community/community#38494] [Other Feature Feedback, Questions, & Ideas]
Hello, when writing a message, each time I use the preview function and go back to writing, I lose the whole edit history.
I can no longer go back to a previous version of the message by using CTRL+Z.

How to reproduce the issue:
- Start a message (like an answer to this post)
- Write a lot of words
- Try CTRL+Z -> it works, the last modification is reversed
- Click on Preview
- The preview is displayed
- Click on Write
- The Write text box is displayed
- Try CTRL+Z -> it does not work, nothing happens

It seems to happen in all Write / Preview text boxes.
Is there a way to correct this, or at least avoid losing the edit history?

**Comment by david-engelmann** [comment]
I'm not currently able to reproduce this. I see the CTRL + Z functionality and preview functionality working correctly!

**Reply by marckruzik** [reply]
I recorded the complete process here, with the bug as described.
We can clearly see that after using the preview, the "undo" option is greyed out:

![anim](https://user-images.githubusercontent.com/51711029/201398724-eff825da-dce0-43e2-9abc-1d858bbf6101.gif)
